dao/pom.xml

```diff
@@ -47,6 +47,10 @@
             <groupId>org.thingsboard.common</groupId>
             <artifactId>dao-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Maybe create queue-api? This may be redundant. What classes from queue module do you need?